lib/src/component_declaration/component_type_checking.dart

```diff
@@ -231,20 +232,27 @@ bool isComponentOfType(ReactElement instance, dynamic typeAlias, {
     return false;
   }
 
+  var finalInstance = instance;
+  var instanceType = finalInstance.type;
+
   var type = getComponentTypeFromAlias(typeAlias);
   if (type == null) {
     return false;
   }
 
-  var instanceType = instance.type;
+  if (instance.type != null && getProperty(instance.type, 'WrappedComponent') != null) {
```

Comment:
This `WrappedComponent` stuff needs some comments. Where does this come from?

Reply:
addressed